code-challenges.test.js

```diff
@@ -24,6 +24,26 @@ const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "s
 
 // Pseudo code:
 
+// const shuffleArray = (array) => {
+// array.shift()
+//  const shuffle = array.sort(() => Math.random())
+//  return shuffle
+// }
+
```

Comment:
This almost works! There's one little tweak you can make and it will work.  See if you can find it  -> js sort method random shuffle